src/mem/qos/q_policy.cc

```diff
@@ -72,6 +72,8 @@ QueuePolicy::create(const QoSMemCtrlParams &p)
 QueuePolicy::PacketQueue::iterator
 LrgQueuePolicy::selectPacket(PacketQueue* q)
 {
+    panic_if(q->empty(),
```

Comment:
The comment on line 142 probably needs an update

Reply:
Thanks, I hadn't noticed it - I've updated the comment to reflect the changes